packages/google_sign_in/google_sign_in_web/example/integration_test/auth2_test.dart

```diff
@@ -10,6 +10,7 @@ import 'package:google_sign_in_platform_interface/google_sign_in_platform_interf
 import 'package:google_sign_in_web/google_sign_in_web.dart';
 import 'gapi_mocks/gapi_mocks.dart' as gapi_mocks;
 import 'src/test_utils.dart';
+import 'package:js/js_util.dart' as js_util;
```

Comment:
Nit: group package imports together before project imports.

Reply:
I'm giving a pass to the whole package. I wish this was something that dartfmt -w did automagically :/